packages/ts-newskit/src/slices/lead-story-2/article-stacks.tsx

```
@@ -25,26 +25,27 @@ export const ArticleStack = ({
   horizontalArticles: LeadArticleProps[];
   horizontalArticleContentWidth?: string;
 }) => {
-  const modifiedVerticalArticles = verticalArticles.map(item => ({
+  const modifiredHorizontalArticles = horizontalArticles.map(item => ({
     ...item,
     imageTop: true,
     headlineTypographyPreset: 'editorialHeadline020',
     hasTopBorder: false
   }));
 
-  const modifiredHorizontalArticles = horizontalArticles.map(item => ({
+  const modifiedVerticalArticles = verticalArticles.map(item => ({
     ...item,
     imageTop: true,
```

Does this want imageTop still?

---

yes it does and the reason for that is default layout of leadArticle is `image right` so when you hideImage it still takes the default layout if `imageTop: true` not provided but I agree if we hide the image we should not need to pass the `imageTop: true` and I will create a tech debt ticket to refactor it